### DISCUSSION TITLE
val_accuracy objective goes for very high epoch count with very small (sub 0.001) incremental improvement.

### DISCUSSION THREAD

**Discussion by istavnit** [keras-team/autokeras#1353] [General]
During the search for network architecture Automodel with objective of val_accuracy gets stuck (see picture). I imagine that happens because min_delta on objective is set to some very small number. Is it possible to provide the min_delta to Automodel so it looks for a minimum amount of increment to keep going?

![image](https://user-images.githubusercontent.com/7511567/94356376-60211b00-0053-11eb-8f9f-68774be712cf.png)

**Comment by haifeng-jin** [comment]
Not sure if you are using 1.0.9.
But this [line](https://github.com/keras-team/autokeras/blob/1.0.9/autokeras/engine/tuner.py#L164) would stop the training if it doesn't decrease enought (>=0.0001) after 10 epochs.
